force-app/main/default/lwc/relationshipsTreeGrid/relationshipsTreeGrid.js

```diff
@@ -105,14 +114,20 @@ export default class RelationshipsTreeGrid extends NavigationMixin(LightningElem
         };
     }
 
-    buildActionsColumn(column) {
+    buildActionsColumn(relationshipsListView) {
+        let rowActions = [ACTION_DEFS.VIEW_RECORD];
+
+        if (relationshipsListView.showCreateRelationshipButton) {
+            rowActions.push(ACTION_DEFS.NEW_RELATIONSHIP);
+        }
+
+        if (this.isLightningOut) {
+            rowActions.push(ACTION_DEFS.RE_CENTER);
```

Comment:
Great solution for this! There's really no need for that button on the Contact detail page.